funnel/explara.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+import requests
+
+__all__ = ['ExplaraAPI']
+
+
+class ExplaraAPI(object):
+    # explara_api = ExplaraAPI({'access_token'}: app.config.get('EXPLARA_ACCESS_TOKEN'))
+    # tickets = explara_api.get_tickets(app.config.get('EXPLARA_EVENT_IDS').get('rootconf-2015'))
+    def __init__(self, config):
+        self.access_token = config.get('access_token')
+
+    def get_orders(self, explara_event_id):
+        headers = {'Authorization': 'Bearer {0}'.format(self.access_token)}
```

Comment:
`''.format` doesn't automatically cast plain strings into Unicode strings if any of the parameters are Unicode, so this can bite. You'll need to start with a Unicode string. Since it's just string concatenation, use this instead: `u'Bearer ' + self.access_token`